class.jetpack.php

```diff
@@ -7258,4 +7261,66 @@ public static function handle_post_authorization_actions( $activate_sso = false,
 
 		Jetpack::state( 'message', 'authorized' );
 	}
+
+	/**
+	 * Load Gutenberg editor blocks
+	 *
+	 * Blocks are disabled by default and enabled via filter:
+	 *   add_filter( 'jetpack_gutenberg', '__return_true', 10 );
+	 *
+	 * When enabled, blocks are loaded from CDN by default. To load locally instead:
+	 *   add_filter( 'jetpack_gutenberg_cdn', '__return_false', 10 );
+	 *
+	 * Note that when loaded locally, you need to build the files yourself:
+	 * - _inc/blocks/jetpack-editor.js
+	 * - _inc/blocks/jetpack-editor.css
+	 *
+	 * CDN cache is busted once a day or when Jetpack version changes. To customize it:
+	 *   add_filter( 'jetpack_gutenberg_cdn_cache_buster', function( $version ) { return time(); }, 10, 1 );
+	 */
+	public static function load_jetpack_gutenberg() {
+		if ( ! apply_filters( 'jetpack_gutenberg', false ) || ! Jetpack::is_plugin_active( 'gutenberg/gutenberg.php' ) ) {
```

Comment:
this check `Jetpack::is_plugin_active( 'gutenberg/gutenberg.php' ) )` will break if gutenberg is not there as a plugin (as will happen on 5.0 and probably nighlty releases of WP 5.0 as soon as somebody commits gutenberg into it(. 

Would it be enough to just check for the availability of `register_block_type` or , as mentioned in my next comment, using a `Jetpack::is_gutenberg_available` handy function ?